libafl/src/corpus/testcase.rs

```diff
@@ -322,25 +288,46 @@ impl<I> Testcase<I>
 where
     I: Input + HasLen,
 {
-    /// Get the cached len
+    /// Get the cached `len`. Will `Error::EmptyOptional` if `len` is not yet cached.
     #[inline]
     pub fn cached_len(&mut self) -> Result<usize, Error> {
```

Comment:
cached_len and load_len is almost the same now.
I think it's better to change cached_len to a simple getter method

Reply:
Good point, done